gateway-util-common/src/test/java/org/apache/knox/gateway/util/UrlsTest.java

```diff
@@ -94,4 +94,9 @@ public void testURLEncoding() throws Exception {
     assertEquals( "%3F", Urls.encode( "?" ) );
   }
 
+  @Test
+  public void testContainsUserInfo() throws Exception {
+    assertEquals(true, Urls.containsUserInfo( "https://www.local.com:8443aa@google.com"));
```

Comment:
nit: assertTrue/assertFalse instead of assertEquals.

Reply:
Update: even the build failed with the same error message. You need to change this.

Reply:
+1